### PR TITLE
fix: ARM images

### PR DESCRIPTION
**Pull request by mateusoliveira43**
Create both AMD and ARM images in images jobs

### CODE REVIEW

**Comment by mateusoliveira43** [comment]
/pj-rehearse pull-ci-openshift-oadp-operator-master-images pull-ci-openshift-oadp-operator-master-unit-test-arm pull-ci-openshift-oadp-operator-master-unit-test

**Comment by mateusoliveira43** [comment]
/pj-rehearse pull-ci-openshift-oadp-operator-master-images pull-ci-openshift-oadp-operator-oadp-1.4-images

File: ci-operator/config/openshift/oadp-operator/.config.prowgen [hunk 1]
```diff
@@ -0,0 +1 @@
+multi_arch: yes
```
Comment (droslean) [inline ci-operator/config/openshift/oadp-operator/.config.prowgen:1]: This is an experimental feature that we are planning to deprecate in the future. You don't have to worry about that. However, this is a configuration for the prowjob generation which it ensure that the job will run on the only cluster we have that holds arm64 nodes. 

You will still have to add the `multi_arch: true` on every of the images in the ci-operator configuration.

Comment (mateusoliveira43) [reply ci-operator/config/openshift/oadp-operator/.config.prowgen:1]: thanks, will add the missing config!

**Comment by mateusoliveira43** [comment]
/pj-rehearse pull-ci-openshift-oadp-operator-master-images pull-ci-openshift-oadp-operator-oadp-1.4-images pull-ci-openshift-velero-plugin-for-aws-konveyor-dev-images pull-ci-openshift-velero-plugin-for-aws-oadp-1.4-images pull-ci-openshift-velero-konveyor-dev-images pull-ci-openshift-velero-oadp-1.4-images

**Comment by mateusoliveira43** [comment]
/pj-rehearse pull-ci-openshift-oadp-operator-master-images pull-ci-openshift-oadp-operator-oadp-1.4-images pull-ci-openshift-velero-plugin-for-aws-konveyor-dev-images pull-ci-openshift-velero-plugin-for-aws-oadp-1.4-images pull-ci-openshift-velero-konveyor-dev-images pull-ci-openshift-velero-oadp-1.4-images

**Comment by mateusoliveira43** [comment]
/pj-rehearse pull-ci-openshift-oadp-operator-master-images pull-ci-openshift-oadp-operator-oadp-1.4-images pull-ci-openshift-velero-plugin-for-aws-konveyor-dev-images pull-ci-openshift-velero-plugin-for-aws-oadp-1.4-images pull-ci-openshift-velero-konveyor-dev-images pull-ci-openshift-velero-oadp-1.4-images

**Comment by mateusoliveira43** [comment]
/pj-rehearse pull-ci-openshift-velero-plugin-for-gcp-konveyor-dev-unit-test pull-ci-openshift-velero-plugin-for-gcp-oadp-1.4-unit-test pull-ci-openshift-velero-konveyor-dev-unit-test pull-ci-openshift-velero-oadp-1.4-unit-test pull-ci-openshift-oadp-operator-master-unit-test pull-ci-openshift-oadp-operator-oadp-1.4-unit-test pull-ci-openshift-velero-plugin-for-aws-konveyor-dev-unit-test pull-ci-openshift-velero-plugin-for-aws-oadp-1.4-unit-test pull-ci-openshift-velero-plugin-for-microsoft-azure-konveyor-dev-unit-test pull-ci-openshift-velero-plugin-for-microsoft-azure-oadp-1.4-unit-test pull-ci-openshift-openshift-velero-plugin-master-unit-test pull-ci-openshift-openshift-velero-plugin-oadp-1.4-unit-test

**Comment by mateusoliveira43** [comment]
/pj-rehearse pull-ci-openshift-oadp-operator-oadp-1.4-4.16-e2e-test-aws pull-ci-openshift-oadp-operator-oadp-1.4-4.16-e2e-test-kubevirt-aws

**Comment by mateusoliveira43** [comment]
/pj-rehearse pull-ci-openshift-velero-plugin-for-gcp-konveyor-dev-unit-test pull-ci-openshift-velero-plugin-for-gcp-oadp-1.4-unit-test pull-ci-openshift-velero-konveyor-dev-unit-test pull-ci-openshift-velero-oadp-1.4-unit-test pull-ci-openshift-velero-plugin-for-aws-konveyor-dev-unit-test pull-ci-openshift-velero-plugin-for-aws-oadp-1.4-unit-test pull-ci-openshift-velero-plugin-for-microsoft-azure-konveyor-dev-unit-test pull-ci-openshift-velero-plugin-for-microsoft-azure-oadp-1.4-unit-test

**Comment by mateusoliveira43** [comment]
/pj-rehearse pull-ci-openshift-velero-plugin-for-gcp-konveyor-dev-unit-test pull-ci-openshift-velero-plugin-for-gcp-oadp-1.4-unit-test pull-ci-openshift-velero-konveyor-dev-unit-test pull-ci-openshift-velero-oadp-1.4-unit-test pull-ci-openshift-velero-plugin-for-aws-konveyor-dev-unit-test pull-ci-openshift-velero-plugin-for-aws-oadp-1.4-unit-test pull-ci-openshift-velero-plugin-for-microsoft-azure-konveyor-dev-unit-test pull-ci-openshift-velero-plugin-for-microsoft-azure-oadp-1.4-unit-test

**Review by sseago — APPROVED** [review]

**Review by kaovilai — APPROVED** [review]

**Pull request merged**